spec/components/degrees_review_component_spec.rb

```diff
@@ -65,4 +67,13 @@
     expect(result.css('.app-summary-card__title').text).to include('BA Woof')
     expect(result.css('.app-summary-card__title').text).to include('BA Meow')
   end
+
+  it 'renders component along with a delete link for each degree' do
+    result = render_inline(DegreesReviewComponent, application_form: application_form)
+
+    expect(result.css('.app-summary-card__actions').text).to include(t('application_form.degree.delete'))
+    expect(result.css('.app-summary-card__actions a')[0].attr('href')).to include(
```

Comment:
maybe you do not need to check the exact URL, since you have a system test that click on that link